--- a/usr/share/YaST2/lib/y2storage/proposal/autoinst_drive_planner.rb
+++ b/usr/share/YaST2/lib/y2storage/proposal/autoinst_drive_planner.rb
@@ -150,6 +150,9 @@
         device.encryption_label = partition_section.crypt_label
         device.encryption_cipher = partition_section.crypt_cipher
         device.encryption_key_size = encryption_key_size_for(partition_section)
+        if device.encryption_method&.is?(:systemd_fde)
+          device.encryption_authentication = proposal_settings.encryption_authentication
+        end
         if device.encryption_method&.is?(:pervasive_luks2)
           device.encryption_pervasive_apqns = apqns_for(partition_section)
           device.encryption_pervasive_key_type = partition_section.crypt_pervasive_key_type
