--- a/usr/share/YaST2/lib/bootloader/bls.rb
+++ b/usr/share/YaST2/lib/bootloader/bls.rb
@@ -34,8 +34,11 @@
       )
     end

-    def self.install_bootloader
-      Yast::Execute.on_target!(SDBOOTUTIL, "install")
+    def self.install_bootloader(portable: false)
+      command = [SDBOOTUTIL]
+      command << "--portable" if portable
+      command << "install"
+      Yast::Execute.on_target!(*command)
     rescue Cheetah::ExecutionFailed => e
       Yast::Report.Error(
       format(_(
--- a/usr/share/YaST2/lib/bootloader/systemdboot.rb
+++ b/usr/share/YaST2/lib/bootloader/systemdboot.rb
@@ -143,7 +143,7 @@
       log.info("Writing settings...")
       write_kernel_parameter
       if Yast::Stage.initial # while new installation only (currently)
-        Bls.install_bootloader
+        Bls.install_bootloader(portable: update_nvram == false)
         Bls.set_authentication
       end
